requirements.txt

```diff
@@ -9,15 +9,15 @@ coreapi==2.3.3
 coreschema==0.0.4
 coverage==4.5.3
 cryptography==2.6.1
+dj-database-url==0.5.0
 Django==2.1
-django-cors-headers==3.0.1
 django-cors-middleware==1.3.1
 django-extensions==2.1.0
-django-filter==2.1.0
```

Comment:
l would like to know why you deleted `django-cors-headers==3.0.1` and `django-filter==2.1.0
 ` from the requirements.txt

Reply:
@jennizalwango, thanks for pointing this out. I guess, I accidently removed these two lines while doing the rebase. let me add them,

Reply:
Good work @alexxsanya  l have actually followed the steps you outlined and everything so far works